### PR TITLE
[DNM] Testing whitebox integration into CI

### PR DESCRIPTION
**Pull request by jamepark4**
Testing the inclusion of whitebox into tempest pod for nova-operator ci.

### CODE REVIEW

File: ci/nova-operator-compute-kit/playbooks/tempest.yaml [hunk 1]
```diff
@@ -22,8 +22,20 @@
...
+      ansible.builtin.command: "oc rsh pod/tempest-worker sudo dnf install -y --nogpgcheck python3-pip"
+      changed_when: true
+
+    - name: Install whitebox requirements
+      ansible.builtin.command: "oc rsh pod/tempest-worker pip install -r /home/stack/whitebox-tempest-plugin/requirements.txt"
+      changed_when: true
+
+    - name: Install whitebox
+      ansible.builtin.command: "oc rsh pod/tempest-worker pip install /home/stack/whitebox-tempest-plugin"
+      changed_when: true
```
Comment (SeanMooney) [inline ci/nova-operator-compute-kit/playbooks/tempest.yaml:35]: you can technically combine these into one call just add the path  to the source dir at after the requirement file.

**Comment by SeanMooney** [comment]
if i remmeber correctly you mentioned else where that whitebox is now included in the tempest image or there was a patch to add it. has that been merged?

**Comment by jamepark4** [comment]
> if i remmeber correctly you mentioned else where that whitebox is now included in the tempest image or there was a patch to add it. has that been merged?

I have not had a chance to try it but, yes it was added to tcib: https://github.com/openstack-k8s-operators/tcib/pull/66

**Comment by SeanMooney** [comment]
we will run whitebox via the test operator so closing this for now.

**Pull request closed without merging**